src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css

```diff
@@ -8,6 +8,11 @@ a, .btn-link {
     color: #0077cc;
 }
 
+.form-control:focus {
+    border-color: #0077cc;
+    box-shadow: 0 0 0 0.2rem #0077cc;
+}
```

Comment:
Can this not be in the Blazor `site.css`/`app.css` files? As far as I know, in the Blazor cases, the identity UI will still be bringing its own separate `.css` file so there should be no need or relevance to change it in the Blazor CSS.

Reasoning: the `site.css` for identity contains a load of rules like `.nav-pills` and `.navbar-brand` that are not mentioned at all in the Blazor CSS files. So identity can't be relying on all its rules coming from the Blazor CSS files.

Does that sound right to you?

Reply:
Update: I just checked and the identity UI does have its own separate CSS file loaded in the Blazor case, so there's no need to duplicate its rules in the Blazor `site.css`/`app.css` files.

Reply:
ok cool, I'll remove these